ui/src/app/api/autumn/[...all]/route.ts

```diff
@@ -1,18 +1,25 @@
-import { autumnHandler } from "autumn-js/next";
+import { autumnHandler } from 'autumn-js/next';
 import { cookies } from 'next/headers';
 import { jwtDecode } from 'jwt-decode';
 
 // Define the type for Cognito JWT claims
 interface CognitoJwtClaims {
   sub: string;
   email: string;
-  'given_name'?: string;
-  'family_name'?: string;
+  given_name?: string;
+  family_name?: string;
   [key: string]: any;
 }
 
 export const { GET, POST } = autumnHandler({
   identify: async (request) => {
+    if (
+      process.env.NEXT_PUBLIC_LOCAL_MODE === 'true' ||
+      process.env.TRACE_ROOT_LOCAL_MODE === 'true'
```

Comment:
Let's remove this for now as `TRACE_ROOT_LOCAL_MODE` is for `rest`
The naming now is a little bit misleading, created a issue for this https://github.com/traceroot-ai/traceroot/issues/75